AAI/AAIConnectProfile.md

```diff
@@ -449,7 +452,9 @@ the Broker.
 
 5.  Claim Clearinghouses MUST provide protection against attacks as outlined in
     [RFC 6819](https://tools.ietf.org/html/rfc6819).
-    
+
+    1. Section 5.1.6 of RFC 6819 contains a SHOULD section that states `Ensure that client applications do not share tokens with 3rd parties.` This profile provides a mechanism for Clearinghouses to consume access tokens from multiple brokers in a manner that does not involve 3rd parties. Client applications should take care to not spread the tokens to any other services that would be considered 3rd parties.
```

Comment:
I'm okay with this wording.

Of course, the best way to do this is to have the Clearinghouse allocate the token on its own behalf with "aud" set to the Clearinghouse, or to be listed in the "aud" set (sometimes "azp") as allocated by another. This makes the token clearly not a 3rd party, as it is intended for the given audience. However, I believe there is more coming on the next PR. Up to you how you want to handle this.

Reply:
I think this is clear on the next PR.